packages/venia-ui/lib/components/CheckoutPage/checkoutPage.js

```diff
@@ -57,25 +57,28 @@ const CheckoutPage = props => {
         setPaymentInformationDone,
         resetReviewOrderButtonClicked,
         handleReviewOrder,
-        reviewOrderButtonClicked
+        reviewOrderButtonClicked,
+        revertPaymentInformationDone
     } = talonProps;
 
     const [, { addToast }] = useToasts();
 
     useEffect(() => {
         if (hasError) {
+            const message =
+                error && error.message
+                    ? error.message
+                    : 'Oops! An error occurred while submitting. Please try again.';
+
             addToast({
                 type: 'error',
                 icon: errorIcon,
-                message:
-                    'Oops! An error occurred while submitting. Please try again.',
+                message,
                 dismissable: true,
                 timeout: 7000
             });
 
-            if (process.env.NODE_ENV !== 'production') {
```

Comment:
Why remove this check?

Reply:
We have `drop-consoles: true` for production env in webpack. It will drop all console.* statements in production and not purge them in development.

Reply:
Yea - I called the same thing out to @zetlen a while back but I can't find the specific comment. I think it's fine to just be explicit with our console logging. It'll get removed in a prod build either way so there's no reason to make this change.

Reply:
I think we shouldn't rely on drop_console because it's a little bit magic, and console logging is not the only kind of development-only code we may write. There's potentially other third-party libraries, or functions with different names, that we'll want to use when showing friendly errors, and the minifier won't catch those.

Reply:
@revanth0212, I think the suggestion here is to add the condition back in.

Reply:
Yeah I have done in it a commit that I haven't pushed yet.

Reply:
Gotcha - I'd just leave the comment unresolved then. When you push it'll resolve it for you and we avoid confusion.

Reply:
The argument that we should add this condition because we can't rely on drop_console is contradictory. If someone removes that flag, I feel like their intention is to see console output, which this would prevent since its behind a condition. Are we 100% on this decision?

Reply:
Nice point @tjwiebell. Haven't thought about it that way. @zetlen any thoughts?